src/features/search/components/QuickSearch.tsx

```diff
@@ -0,0 +1,12 @@
+import AdvancedSearch from "./AdvancedSearch";
+
+const QuickSearch = () => {
+  return (
+    <div className="grow py-32">
+      <h1 className="text-lg font-semibold pt-5  px-4">Quick Search</h1>
+      <AdvancedSearch />
```

Comment:
Is this quick search's functionality only to call the advanced search?

Reply:
Thank you. I forgot about that.

Reply:
I will make changes on that, thank you.